File: 002-Backend/LaPlata.API/LaPlata.Domain/Models/Despesas/Despesa.cs
using System.ComponentModel.DataAnnotations;

namespace LaPlata.Domain.Models
{
    public class Despesa : BaseModel
    {
        [Required]
        public string Descricao { get; set; }
        [Required]
        public int ValorInteiro { get; set; }
        public int? ValorCentavos { get; set; }
        [Required]
        public DateTime DataDespesa { get; set; }
        public int? QtdParcelas { get; set; }
        public virtual Categoria Categoria { get; set; }
        [Required]
        public int CategoriaId { get; set; }
    }
}

File: 003-Frontend/HomeTools.Front/src/app/interfaces/financas/Despesa.ts
import { Categoria } from "../categoria";

export interface Despesa {
  Id: number;
  Descricao: string;
  ValorInteiro: number;
  ValorCentavos: number;
  DataDespesa: Date;
  QtdParcelas: number;
  Categoria: Categoria;
}
